observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerExtension.java

```diff
@@ -4,15 +4,24 @@
  */
 package org.wildfly.extension.micrometer;
 
+import org.jboss.as.version.Stability;
 import org.wildfly.subsystem.SubsystemConfiguration;
 import org.wildfly.subsystem.SubsystemExtension;
 import org.wildfly.subsystem.SubsystemPersistence;
 
 public class MicrometerExtension extends SubsystemExtension<MicrometerSubsystemSchema> {
+    static final Stability FEATURE_STABILITY = Stability.DEFAULT;
```

Comment:
This is only used, as far as I can tell, in this class. It could be private. 

Also, should his be default or community? I'm not certain on this, but I just see other references to community.